src/formatter.rs

```diff
@@ -109,89 +108,101 @@ fn clean_line_size(line: &str) -> usize {
     line.len() - 4 * amount_tag_blocks
 }
 
-/// Adds reverse nester tags
-/// ```
-///  <<Next   [[N]]     {{3}}<< lines:>>
-///  <<Next >>[[N]]<< >>{{3}}<< lines:>>
-/// ```
-/// NOTE: Only supports some combinations.
-///
-/// NOTE: Hacky. Based upon the fact that only 1-level nesting makes sense.
-fn format_add_reverse_nester_tags(text: &str) -> String {
-    // <<Next [[N]] {{3}} lines:>>
-    let mut result = RE_BOLD
-        .replace_all(&text, |caps: &regex::Captures| {
-            let escaped_vars = RE_VARIABLE
-                .replace_all(&caps[0], |inner_caps: &regex::Captures| format!(">>{}<<", &inner_caps[0]))
-                .to_string();
-            RE_CONSTANT
-                .replace_all(&escaped_vars, |inner_caps: &regex::Captures| format!(">>{}<<", &inner_caps[0]))
-                .to_string()
-        })
-        .to_string();
+fn format_paint(text: &str, ostyle: &OutputStyle) -> String {
+    // FIX: https://www.codingame.com/contribute/view/1783dda5b69105636695dc5bf51de1baf5d0
 
-    // `Next [[N]] {{3}} <<B>> lines:`
-    result = RE_MONOSPACE
-        .replace_all(&result, |caps: &regex::Captures| {
-            let escaped_vars = RE_VARIABLE
-                .replace_all(&caps[0], |inner_caps: &regex::Captures| format!("`{}`", &inner_caps[0]))
-                .to_string();
-            let escaped_cons = RE_CONSTANT
-                .replace_all(&escaped_vars, |inner_caps: &regex::Captures| format!("`{}`", &inner_caps[0]))
-                .to_string();
-            RE_BOLD
-                .replace_all(&escaped_cons, |inner_caps: &regex::Captures| format!("`{}`", &inner_caps[0]))
-                .to_string()
-        })
-        .to_string();
+    // OTHERS
+    //      https://www.codingame.com/contribute/view/750741cba87bb6a6ac8daf5adbe2aa083e24
+    //      https://www.codingame.com/contribute/view/83316b323da5dba40730dbca5c72b46ccfc9
 
-    // {{Next [[N]] `Mono \n and more` lines}}
-    result = RE_CONSTANT
-        .replace_all(&result, |caps: &regex::Captures| {
-            let escaped_cons = RE_VARIABLE
-                .replace_all(&caps[0], |inner_caps: &regex::Captures| {
-                    format!("{}{}{}", "}}", &inner_caps[0], "{{")
-                })
-                .to_string();
-            RE_MONOSPACE
-                .replace_all(&escaped_cons, |inner_caps: &regex::Captures| {
-                    format!("{}{}{}", "}}", &inner_caps[0], "{{")
-                })
-                .to_string()
+    // Hack for all tags to be two chars long
+    let text = RE_MONOSPACE
+        .replace_all(&text, |caps: &regex::Captures| {
+            // Another hack: Quickfix for padding
+            let hack_padding = &caps[1].replace('\n', "$$\n@@");
+            format!("@@{}$$", hack_padding)
```

Comment:
Really not a fan of this hack. It's not clear to me why this `replace_all` step is necessary.

Reply:
The replace is just adding reverse monospace tags around newlines, similar to what we had before. I did change those arbitrary @@ $$ back into two ``.